Interni/NormeProgetto/components/norme-c2.tex

```diff
@@ -0,0 +1,205 @@
+Il processo di fornitura è composto dalle seguenti fasi:
+\begin{itemize}
+	\item avvio;
+	\item approntamento di risposte alle richieste;
+	\item contrattazione;
+	\item pianificazione;
+	\item esecuzione e controllo;
+	\item revisione e valutazione;
+	\item consegna e completamento.
+\end{itemize}
+\subsubsection{Descrizione}
+Il documento include le norme che i membri del gruppo \Gruppo{} devono fare riferimento al fine di ottenere il ruolo di fornitori del proponente \proponente{} e dei committenti \textit{Prof. Tullio Vardanega} e \textit{Prof. Riccardo Cardin}. I membri del gruppo sono tenuti ad attenersi a quanto scritto nella suddetta sezione durante tutte le fasi di sviluppo del prodotto \progetto{}.
+\subsubsection{Aspettative}
+Durante lo svolgimento del progetto il gruppo intende mantenere un dialogo costante con il proponente, in modo da instaurare un rapporto di collaborazione e ricevere un riscontro proficuo sul lavoro svolto.  In particolare, si vuole:
+\begin{itemize}
+	\item determinare gli aspetti chiave che il prodotto dovrà soddisfare;
+	\item definire vincoli e requisiti riguardanti i processi;
+	\item stimare costi e tempistiche di lavoro;
+	\item accordarsi sulla qualifica del prodotto.
+\end{itemize}
+\subsubsection{Studio di fattibilità}
+Il \RdP{} ha il compito di organizzare riunioni tra i membri in modo da promuovere lo lo scambio di opinioni sui capitolati proposti. Una volta che viene trovato un capitolato di interesse comune, gli \anas{} provvedono alla stesura dello \SdF{} che, per ogni capitolato, indica:
+\begin{itemize}
+	\item \textbf{Informazioni generali}: vengono elencate le informazioni di base,  quali nome del progetto, il proponente e il committente;
+	\item \textbf{Descrizione e finalità del capitolato}: sintesi del prodotto, comprendente le caratteristiche principali che vengono richieste e l'obiettivo da raggiungere;
+	\item \textbf{Tecnologie interessate}: elenco delle tecnologie richieste per lo svolgimento del prodotto;
+	\item \textbf{Aspetti positivi, criticità e fattori di rischio}: esposizione delle considerazioni fatte dal gruppo riguardanti gli aspetti positivi e sulle criticità del capitolato in questione;
+	\item \textbf{Conclusioni}: esposizione delle ragioni per le quali il gruppo accetta o rifiuta il capitolato.
+\end{itemize}
+\subsubsection{Altra documentazione fornita}
+In questa sezione vengono elencati i documenti che sono stati forniti all'azienda \proponente{} e ai committenti \textit{Prof. Tullio Vardanega} e \textit{Prof. Riccardo Cardin}. La suddetta documentazione risulta essenziale in modo da tracciare le attività  svolte dal gruppo.
```

Comment:
Cambiare
"...  risulta essenziale in modo da tracciare le attività  svolte dal gruppo ..."
in
"...  risulta essenziale per il tracciamento delle attività svolte dal gruppo ..."

Reply:
Fixed